tests/integration_tests/lightgbm_tuner_tests/test_optimize.py

```diff
@@ -726,6 +726,7 @@ def test_optuna_callback(self) -> None:
 
         assert callback_mock.call_count == 10
 
+    @pytest.mark.skip(reason="Fail since 28 Jan 2024.")
```

Comment:
Could you add a TODO comment?

Reply:
I added a comment to the corresponding two lines.